src/redmodding.ts

```diff
@@ -131,7 +131,7 @@ const prepareForModdingWithREDmodding = async (
   }
 
   const gameStoreIfInstalledThroughStore =
-    await VortexUtil.GameStoreHelper.findByAppId([GOGAPP_ID, STEAMAPP_ID, EPICAPP_ID]);
+    await VortexUtil.GameStoreHelper.findByAppId([GOGAPP_ID, STEAMAPP_ID, EPICAPP_ID]).catch(() => undefined);
 
   if (gameStoreIfInstalledThroughStore?.gamePath !== discovery.path) {
     vortexApi.log(`warn`, `Cyberpunk discovery doesn't match auto-detected path`, { discovery: discovery.path, autoDetect: gameStoreIfInstalledThroughStore.path });
```

Comment:
```suggestion
    vortexApi.log(`warn`, `Cyberpunk discovery doesn't match auto-detected path`, { discovery: discovery.path, gameStoreIfInstalledThroughStore });
```
This is the other thing